src/server/game/AuctionHouseBot/AuctionHouseBotSeller.cpp

```diff
@@ -708,15 +708,15 @@ void AuctionBotSeller::SetPricesOfItem(ItemTemplate const* itemProto, SellerConf
 
     if (sAuctionBotConfig->GetConfig(CONFIG_AHBOT_BUYPRICE_SELLER))
         buyPrice = sellPrice;
+	
+	float basePriceFloat = (buyPrice * stackCount * priceRatio) / (itemProto->GetClass() == 6 ? 200.0f : static_cast<float>(itemProto->GetBuyCount())) / 100.0f;
+    float range = basePriceFloat * 0.04f;
+	
+    buyp = urand(static_cast<uint32>(basePriceFloat - range + 0.5f), static_cast<uint32>(basePriceFloat + range + 0.5f)) + 1;
```

Comment:
Replacing with frand and then converting to int is a good alternate.  The net effect will be similar.